internal/consensus/pbts_test.go

```diff
@@ -297,6 +315,7 @@ func (p *pbtsTestHarness) run() resultSet {
 	startTestRound(p.ctx, p.observedState, p.currentHeight, p.currentRound)
 
 	r1 := p.observedValidatorProposerHeight(p.genesisTime)
+	p.height1Time = r1.proposalIssuedAt
```

Comment:
I think it's slightly preferable to have `observedValidatorProposerHeight` return a second value which is the timestamp of the block. `r1.proposalIssuedAt` should be very close to that time, but no reason not to just grab the actual value from the block instead.